### PR TITLE
Add Liquichain

### PR DESCRIPTION
**Pull request by smichea**
replace deprecated chain by liquichain

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
you need to add redFlags reusedChainId

**Comment by smichea** [comment]
@ligi should i do something else ? thank you

**Review by ligi — CHANGES_REQUESTED** [review]
The explorer must be an object - please see the other chains.

Also: Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by smichea** [comment]
sorry for that @ligi , hope all is fine now

File: _data/chains/eip155-1662.json [hunk 1]
```diff
@@ -12,6 +12,12 @@
   "infoURL": "https://liquichain.io/",
   "chainId": 1662,
   "networkId": 1662,
-  "explorers": ["https://mainnet.liquichain.io/"],
+  "explorers": [
+    {
+      "name": "Liquichain Mainnet",
+      "url": "https://mainnet.liquichain.io/",
```
Comment (ligi) [inline _data/chains/eip155-1662.json:18]:
```suggestion
      "url": "https://mainnet.liquichain.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — APPROVED** [review]

**Pull request merged**